docs/tutorial/electron-timelines.md

```diff
@@ -79,6 +73,36 @@ and the version prior to that receives the vast majority of those fixes
 as time and bandwidth warrants. The oldest supported release line will receive
 only security fixes directly.
 
+### Chromium Version Support
+
+:::info Chromium release schedule
+
+Chromium's public release schedule is [here](https://chromiumdash.appspot.com/schedule).
+
+:::
+
+Electron targets Chromium even-number versions, releasing every 8 weeks in concert
+with Chromium's 4-week release schedule.
+
+### Node.js Version Support
+
+Electron upgrades Node.js to even-number versions when they enter Active LTS. The schedule
```

Comment:
I think there's a phrasing improvement to be made about when the Node.js LTS version upgrade happens.

When a new Chromium stable drops, an Electron stable version with the corresponding Chromium version will also be available. With Node.js, only the `main` branch gets updated at that point in time, right?

Reply:
Edited it - i think it should be a bit clearer now!